env.go

```diff
@@ -123,6 +123,11 @@ func cliColorForced(env environ) bool {
 	return cliColorForce
 }
 
+func envTTYForced(env environ) bool {
```

Comment:
maybe `isTTYForced`?

Reply:
what do you think about `envIsTTYForced`? to keep `env` prefix pattern

Reply:
ah color doesn't have env prefix lol